Dockerfile

```diff
@@ -88,6 +88,8 @@ WORKDIR /app
 
 COPY --from=assets-precompile /app /app
 COPY --from=assets-precompile /usr/local/bundle/ /usr/local/bundle/
+RUN echo export PATH=/usr/local/bin:\$PATH > /root/.ashrc
```

Comment:
not sure this is the best way to do this, see PR description for the other options/examples

Reply:
Isn't it the case that if `ENV PATH="/usr/local/bin:$PATH"` can be overriden, this can too? ie something's clobbering `$PATH`?